soh/soh/GameMenuBar.cpp

```diff
@@ -1099,6 +1103,8 @@ namespace GameMenuBar {
                 UIWidgets::PaddedEnhancementCheckbox("Bombchus out of bounds", "gBombchusOOB", true, false);
                 UIWidgets::Tooltip("Allows bombchus to explode out of bounds\nSimilar to GameCube and Wii VC");
                 UIWidgets::PaddedEnhancementCheckbox("Restore old Gold Skulltula cutscene", "gGsCutscene", true, false);
+                UIWidgets::EnhancementCheckbox("Restore save confirmation", "gSaveConfirmation");
+                UIWidgets::Tooltip("Restores the N64 save confirmation screen");
```

Comment:
```suggestion
                UIWidgets::EnhancementCheckbox("Skip save confirmation", "gSkipSaveConfirmation");
                UIWidgets::Tooltip("Skip the \"Game saved.\" confirmation screen");
```

also move this up to line 770ish with the time savers